manifests/init.pp

```diff
@@ -6,23 +6,25 @@
 # Richard Hillmann <richie@project0.de>
 #
 class bareos (
-  $config_dir       = $bareos::params::config_dir,
-  $config_dir_webui = $bareos::params::config_dir_webui,
-  $file_owner       = $bareos::params::file_owner,
-  $file_group       = $bareos::params::file_group,
-  $file_mode        = $bareos::params::file_mode,
-  $file_dir_mode    = $bareos::params::file_dir_mode,
-  $repo_release     = $bareos::params::repo_release,
-  $manage_repo      = $bareos::params::manage_repo,
-  $manage_user      = $bareos::params::manage_user,
-  $manage_package   = $bareos::params::manage_package,
-  $manage_service   = $bareos::params::manage_service,
-  $manage_database  = $bareos::params::manage_database,
-  $package_name     = $bareos::params::package_name, # base/common package only
-  $package_ensure   = $bareos::params::package_ensure,
-  $service_ensure   = $bareos::params::service_ensure,
-  $service_enable   = $bareos::params::service_enable,
-
+  $config_dir            = $bareos::params::config_dir,
+  $config_dir_webui      = $bareos::params::config_dir_webui,
+  $file_owner            = $bareos::params::file_owner,
+  $file_group            = $bareos::params::file_group,
+  $file_mode             = $bareos::params::file_mode,
+  $file_dir_mode         = $bareos::params::file_dir_mode,
+  $repo_release          = $bareos::params::repo_release,
+  $repo_subscription     = $bareos::params::repo_subscription,
+  $repo_username         = $bareos::params::repo_username,
+  $repo_password         = $bareos::params::repo_password,
+  $manage_repo           = $bareos::params::manage_repo,
+  $manage_user           = $bareos::params::manage_user,
+  $manage_package        = $bareos::params::manage_package,
+  $manage_service        = $bareos::params::manage_service,
+  $manage_database       = $bareos::params::manage_database,
+  $package_name          = $bareos::params::package_name, # base/common package only
+  $package_ensure        = $bareos::params::package_ensure,
+  $service_ensure        = $bareos::params::service_ensure,
+  $service_enable        = $bareos::params::service_enable,
```

Comment:
can you please add datatypes for the new parameters here? And since their value in params.pp is static, please directly assign it here and not in params.pp. Our long term goal is to have puppet-strings documentation for all modules where static data is directly assigned in the init class and the rest comes from data in modules.

Reply:
so the idea is to get of the params.pp, right? then I'll keep that in mind for my coming changes

I took all the `package` and `manage` values into init.pp with their values. Is it good that way?